### PR TITLE
Update German translation majorly

### PR DESCRIPTION
**Pull request by DaCoolX**
Brought up to speed of the template including but not limited to:
All source comments updated
All new and missing translations added
Orphaned entries removed
Appropriate copyright notices added
Additionally:
Minor typos and awkward translations corrected

### CODE REVIEW

File: resources/l10n/de/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -1,584 +1,742 @@
...
+"This appears to be the first time you have run LOOT v%s. Here are some tips "
+"to help you get started with the interface."
+msgstr ""
+"Dies scheint das erste Mal zu sein das Sie LOOT v%s gestartet haben. "
+"Hier sind einige Tipps die Ihnen helfen "
+"Sie mit der Benutzeroberfläche vertraut zu machen"
+
+#: src/gui/html/js/translateStaticText.js:614
+msgid "Click %(menu_icon)s buttons to open menus."
+msgstr "Klicke die %(menu_icon) Schaltfläche um Menüs zu öffnen "
```
Comment (Ortham) [inline resources/l10n/de/LC_MESSAGES/loot.po]: This needs to preserve the whole of `%(menu_icon)s`, the trailing s is important.

Comment (DaCoolX) [reply resources/l10n/de/LC_MESSAGES/loot.po]: Figures that something slips by

**Comment by DaCoolX** [comment]
Done

**Comment by Ortham** [comment]
👍

**Pull request merged**